skpro/distributions/multivariatenormal.py

```diff
@@ -0,0 +1,88 @@
+# copyright: skpro developers, BSD-3-Clause License (see LICENSE file)
+"""Multivariate Normal probability distribution."""
+
+__author__ = ["HarshvirSandhu"]
+
+import numpy as np
+import pandas as pd
+from scipy.stats import multivariate_normal, rv_continuous
+
+from skpro.distributions.adapters.scipy import _ScipyAdapter
+
+
+class MultiVariate_Normal(_ScipyAdapter):
+    r"""Multivariate Normal distribution, aka log-logistic distribution.
+
+    Most methods wrap ``scipy.stats.multivariate_normal``.
+
+    The mean keyword specifies the mean and cov specifies the covariance matrix.
+    The probability distribution function (PDF) is given by:
+
+    .. math::
+
+        f(x) = \frac{1}{\sqrt{(2 \pi)^k \det \Sigma}}
+               \exp\left( -\frac{1}{2} (x - \mu)^T \Sigma^{-1} (x - \mu) \right),
+
+    where :math:`\mu` is the mean, :math:`\Sigma` the covariance matrix,
+    :math:`k` the rank of :math:`\Sigma`.
+
+    Parameters
+    ----------
+    mean : array_like, default: ``[0]``
```

Comment:
```suggestion
    mean : array_like, default = ``[0]``
```
you are using another style  at the lines 35/36. Which is the correct one?

Other question: From the implementation, it seems that mean has not a default value. Or ?

Reply:
Yes, it should be `=` instead of `:`